topics/imaging/tutorials/voronoi-segmentation/tutorial.md

```diff
@@ -0,0 +1,348 @@
+{: .comment}
+
+
+## Getting data from Zenodo
+> <hands-on-title> Data Upload </hands-on-title>
+>
+> 1. Create a new history for this tutorial. When you log in for the first time, an empty, unnamed history is created by default. You can simply rename it.
+> 
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+> 
+> 2. Import {% icon galaxy-upload %} the following dataset from [Zenodo](https://zenodo.org/records/15172302). 
+> 
+>    ```
+>    https://zenodo.org/records/15281843/files/images_and_seeds.zip
+>    ```
+> 
+>    - **Important:** Choose the type of data as `zip`.
+> 
+>    The upload might take a few minutes. 
+> 
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+> 3. {% tool [Unzip](toolshed.g2.bx.psu.edu/repos/imgteam/unzip/unzip/6.0+galaxy0) %} the image with the following parameters:
+>    - {% icon param-file %} *"input_file"*: `images_and_seeds.zip`
+>    - *"Extract single file"*: `Single file`
+>    - *"Filepath"*: Choose which data you want to use: 
+>        - Cells: `images_and_seeds/cell_image-B2--W00026--P00001--Z00000--T00000--dapi.tiff`
+>        - Trees: `images_and_seeds/tree_image_2019_DELA_5_423000_3601000.tiff`
+>    
+> 4. Rename {% icon galaxy-pencil %} the resulting file as `image`.
+>
+> 5. Check that the datatype is correct.
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
+>
+> 6. {% tool [Unzip](toolshed.g2.bx.psu.edu/repos/imgteam/unzip/unzip/6.0+galaxy0) %} the seed with the following parameters:
+>    - {% icon param-file %} *"input_file"*: `images_and_seeds.zip`
+>    - *"Extract single file"*: `Single file`
+>    - *"Filepath"*: Choose the seed image corresponding to the image you chose in the last step. 
+>        - Cells: `images_and_seeds/cell_seeds-B2--W00026--P00001--Z00000--T00000--dapi.tiff`
+>        - Trees: `images_and_seeds/tree_seeds_2019_DELA_5_423000_3601000.tiff`
+>
+> 7. Rename {% icon galaxy-pencil %} the resulting file as `seeds`.
+{: .hands_on}
+
+
+## Generate an object mask from pixel intensity
+In case there should be empty regions without cells in the image, we wish to constrain the single Voronoi regions to roughly the area where a cell is. 
+Therefore, we first smooth the image to reduce the influence of noise, and then apply a threshold on the smoothed image to get a binary mask. 
+> <comment-title> Mask vs seeds </comment-title>
+> The process used to create a mask can also be used to make seeds, as in the 
+> [Imaging introduction tutorial]({% link topics/imaging/tutorials/imaging-introduction/tutorial.md %}). 
+{: .comment}
+
+> <hands-on-title> Task description </hands-on-title>
+> The image has three channels (red, green, blue). To generate a mask, we have to select a channel, for instance channel `0`. 
+> 1. {% tool [Convert image format](toolshed.g2.bx.psu.edu/repos/imgteam/bfconvert/ip_convertimage/6.7.0+galaxy3) %} with the following parameters:
+>    - {% icon param-file %} *"Input Image"*: `image`
```

Comment:
I think an info on the `Output data type` should be added here (probably `TIFF`).